lib/FusionInventory/Agent/Task/Inventory/Virtualization/Xen.pm

```diff
@@ -8,7 +8,7 @@ use FusionInventory::Agent::Tools;
 our $runMeIfTheseChecksFailed = ["FusionInventory::Agent::Task::Inventory::Virtualization::Libvirt"];
 
 sub isEnabled {
-    return canRun('xm');
+    return canRunOK('xm') || canRunOK('xl');
 }
```

Comment:
In this case, no.

Xen >=4.1 let you choose between the old and the new toolstack, and installs a wrapper around both xm and xl, failing if it mismatch the configured toolstack.

So in this case : xm and xl both exist, but only one of them is actually usable.